pkg/local_object_storage/engine/lock_test.go

```diff
@@ -133,7 +134,7 @@ func TestLockUserScenario(t *testing.T) {
 	inhumePrm.WithTarget(tombForLockAddr, lockerAddr)
 
 	_, err = e.Inhume(inhumePrm)
-	require.NoError(t, err, new(apistatus.ObjectLocked))
```

Comment:
1. that test was skipped before (partly because of incorrect behavior)
2. i guess, it was a mistake (it contained `new(apistatus.ObjectLocked)` i do not know why)
3. in fact there should not have been an error according to the previous understanding of the issue (removing a lock object was correct) but not it is prohibited and should return the error